pyaedt/hfss.py

```diff
@@ -47,7 +49,14 @@ class Hfss(FieldAnalysis3D, object):
         designs are present, an empty design is created.
     solution_type : str, optional
         Solution type to apply to the design. The default is
-        ``None``, in which case the default type is applied.
+        ``None``, in which case the user-defined
+        default type is applied.
+        Options are:
+        - "Terminal"
+        - "Modal"
+        - "SBR+"
+        - "Transient"
+        - "Eigenmode"
```

Comment:
```suggestion
        default type is applied. Options are:
    
        - "Terminal"
        - "Modal"
        - "SBR+"
        - "Transient"
        - "Eigenmode"
    
```
To render in the documentation as a list, a blank line must precede the first list item and follow the last list item.